Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3
+
+RUN pip install --upgrade pip
+
+WORKDIR /app
+
+COPY . .
+
+RUN pip install -r requirements.txt
```

Comment:
This is unnecessary (As much as the requirement file), because doing simply `pip install` will install the bot dependency just fine, thank to Poetry.

Reply:
It is necessary for docker, as poetry isn't inside the docker env

Reply:
There's no need for poetry inside docker, you can just do `pip install` if `pyproject.toml`/`poetry.lock` is there

Reply:
No you can't, I tried